include/deal.II/base/time_stepping.h

```diff
@@ -72,6 +77,27 @@ namespace TimeStepping
      * Classical fourth order Runge-Kutta method.
      */
     RK_CLASSIC_FOURTH_ORDER,
+    /**
+     * Three-stage scheme of order three by Kennedy et al.
+     * @cite KennedyCarpenterLewis2000 While its stability region is
+     * significantly smaller than for the other schemes, it only involves three
+     * stages, so it is very competitive in terms of the work per stage.
```

Comment:
Can you slightly rephrase this:
```suggestion
     * @cite KennedyCarpenterLewis2000 Its stability region is
     * significantly smaller than for the higher order schemes, but due to three
     * stages only, it is very competitive in terms of the work per stage.
```